pkg/apis/postgres-operator.crunchydata.com/v1beta1/standalone_pgadmin_types.go

```diff
@@ -42,6 +42,14 @@ type StandalonePGAdminConfiguration struct {
 	// +kubebuilder:validation:Schemaless
 	// +kubebuilder:validation:Type=object
 	Settings SchemalessObject `json:"settings,omitempty"`
+
+	// OauthConfigurations allows the user to reference one or more Secrets
+	// containing OAUTH2 configuration settings for pgAdmin.
+	// Each Secret shall contain a single data key called oauth-config.json
+	// whose value is a JSON object containing the OAUTH2 configuration settings.
+	// More info: https://www.pgadmin.org/docs/pgadmin4/latest/oauth2.html
+	// +optional
+	OauthConfigurations []corev1.LocalObjectReference `json:"oauthConfigurations,omitempty"`
```

Comment:
I think it's probably fine to tell the user that they have to name the file a specific thing; however, if we wanted to allow the user to name it whatever they want, you could use `SecretKeyRef` found in `pkg/apis/postgres-operator.crunchydata.com/v1beta1/config_types.go`